arcsi/api/item.py

```diff
@@ -59,6 +59,12 @@ class ItemDetailsSchema(Schema):
         ),
         required=True,
     )
+    tags = fields.List(
```

Comment:
Nice, we can use tags for episodes in the future as well. This will open a lot of new opportunities for the listeners to surf between episodes of different shows and give some interesting, new content for the show and episode subpages too.